nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/provider/opencensusprovider.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {tracer} = require('@opencensus/nodejs');
+const {toW3CTraceContext} = require('../util');
+
+exports.OpenCensusProvider = class OpenCensusProvider {
+    getW3CTraceContext() {
+        if (tracer.active) {
+            const carrier = {};
+            toW3CTraceContext(tracer.currentRootSpan, carrier);
+            return carrier;
+        } else {
```

Comment:
To make this slightly more readable, could you pull both the carried declaration and return value out of the if-else, and eliminate the else?